src/Containers/Chair.tsx

```diff
@@ -10,65 +13,76 @@ export interface IChair {
      * Boolean value for whether someone is seated in the chair
```

Comment:
No need to say Boolean value, it's implied by typescript type this is repetition